src/_ert_forward_model_runner/reporting/event.py

```diff
@@ -74,9 +72,10 @@ def __init__(self, evaluator_url, token=None, cert_path=None):
 
         self._ens_id = None
         self._real_id = None
-        self._event_queue = queue.Queue()
+        self._event_queue: queue.Queue[events.Event | Type[Event._sentinel]] = (
```

Comment:
Is it safe to use `_sentintel = None` instead of object()?

Reply:
yes. only _dump_event and finished_handler will put anything in it. _dump_event only puts Event objects in the queue while finished_handler will put Event._sentinel which is None.